src/rpc/blockchain.cpp

```diff
@@ -32,7 +31,6 @@
 #include <warnings.h>
 
 #include <assert.h>
-#include <stdint.h>
```

Comment:
nit: you should leave these includes
> Every .cpp and .h file should #include every header file it directly uses classes, functions or other definitions from, even if those headers are already included indirectly through other headers.
https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#source-code-organization

Reply:
Fixed.